common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/IndexStatus.kt

```diff
@@ -12,7 +12,6 @@ import java.time.LocalDateTime
 const val INDEX_STATUS_ID = "STATUS"
 
 enum class IndexState(val active: Boolean) {
-  ABSENT(false),
```

Comment:
I kept this class as it still keeps track of whether a build is in progress, even though it is only refreshing